app/src/main/java/com/example/travellens/MessageAdapter.java

```diff
@@ -0,0 +1,114 @@
+package com.example.travellens;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.parse.ParseUser;
+
+
+import java.util.List;
+
+public class MessageAdapter extends RecyclerView.Adapter<MessageAdapter.MessageViewHolder> {
+    public static final int MESSAGE_LEFT = 0; // incoming
+    public static final int MESSAGE_RIGHT = 1; // outcoming
```

Comment:
I would suggest renaming these to INCOMING_MESSAGE_TYPE and OUTGOING_MESSAGE_TYPE so it's more obvious what they represent.